Source/Shaders/Builtin/Constants/passCesium3DTileClassificationIgnoreShow.glsl

```diff
@@ -0,0 +1,9 @@
+/**
+ * The automatic GLSL constant for {@link Pass#CESIUM_3D_TILE_CLASSIFICATION_IGNORE_SHOW}
+ *
+ * @name czm_passCesium3DTileClassificationIgnoreShow
+ * @glslConstant
+ *
+ * @see czm_pass
+ */
+const float czm_passCesium3DTileClassificationIgnoreShow = 6.0;
```

Comment:
Is this used anywhere?

Reply:
No. The only `czm_pass*` constant that is used is `czm_passTranslucent`.

Reply:
So should this be removed?

Reply:
These constants mirror the `Pass` enum. I'm OK with removing this because it's not used, but then why not remove the rest?